modules/kafka/README.md

```diff
@@ -43,19 +43,46 @@ installed, compile with `--with-librdkafka`.
 Running Kafka
 -------------
 
-If you are not too familiar with Kafka, a simple recipe using
-[Docker](http://docker.io) can get you started in a minute:
+If you are not too familiar with Kafka, a simple recipe
+[Kafka](https://kafka.apache.org/quickstart) can get you started in a minute:
 
+
+[Download](https://www.apache.org/dyn/closer.cgi?path=/kafka/2.5.0/kafka_2.12-2.5.0.tgz) the 2.5.0 release and un-tar it
+```
+tar -xzf kafka_2.12-2.5.0.tgz
+cd kafka_2.12-2.5.0
+```
+
+Start the ZooKeeper server once inside kafka_2.12-2.5.0 directory
```

Comment:
1. imho it is better not to replicate the content of the kafka quickstart here, link should be enough.
2. please keep kafkacat reference, that is a nice tool to use (you can just keep that oneliner about how to use it, everybody should be able to obtain it)

just out of curiosity, the `spotify/kafka` docker image are gone, or just does not build? (I would not build it myself if it is possible to just pull from docker.io)

Reply:
@konkan there are issues with building spotify/kafka directly and also with building it from docker image. It is unable to fetch some files. 
On the other hand, I found the official kafka guide to be much more simpler and easy to follow

Reply:
I agree with @Kokan . Let's just use a reference link. There are fix versions, download links that we should followup, and port back the changes when the quickstart guide changes.